docker-compose.yml

```diff
@@ -172,19 +174,19 @@ services:
                 limits:
                     memory: 2G
 
-    nginx:
-        build: https://github.com/fecgov/fecfile-api-proxy.git#develop
-        container_name: fecfile-api-proxy
-        sysctls:
-            - net.ipv6.conf.all.disable_ipv6=1
-        ports:
-            - 8080:8080
-        depends_on:
-            - api
-        deploy:
-            resources:
-                limits:
-                    memory: 256M
```

Comment:
nginx was smacking down all my upload requests because the requests were too large.  Instead of taking the time to actually fix the problem, I just turned off nginx for the sake of the proof-of-concept